README.md

```diff
@@ -12,16 +12,60 @@ Quick setup
 
 0. Install the Ruby SDK with `gem`
 
-        gem install ldclient-rb
+```shell
+gem install ldclient-rb
+```
 
 1. Require the LaunchDarkly client:
 
-        require 'ldclient-rb'
-
+```ruby
+require 'ldclient-rb'
+```
 
 2. Create a new LDClient with your API key:
 
-        client = LaunchDarkly::LDClient.new("your_api_key")
+```ruby
+client = LaunchDarkly::LDClient.new("your_api_key")
+```
+
+### Ruby on Rails
+
+0.  Add `gem 'ldclient-rb'` to your Gemfile and `bundle install`
+
+1.  Initialize the launchdarkly client in `config/initializers/launchdarkly.rb`:
+
+```ruby
+Rails.configuration.ld_client = LaunchDarkly::LDClient.new("your_api_key")
+```
+
+2.  You may want to include a function in your ApplicationController
+
+```ruby
+    def launchdarkly_settings
+      if current_user.present?
+        {
+          key: current_user.id,
+          anonymous: false,
+          email: current_user.email,
+          custom: { groups: current_user.groups.pluck(:name) },
+          # Any other fields you may have
+          # e.g. lastName: current_user.last_name,
+        }
+      else
+        hash_key = UUIDTools::UUID.random_create.to_s
```

Comment:
We just need to be sure this random uuid gets set on the session (and read from there, instead of generating a new one, if it already exists).